### PR TITLE
[WIP] docs: improve linux tar packages

### LINKED ISSUE #8160: Binary release tarballs should come with a README
A typical release tarball (or zipball) currently contains the following:

```
bitcoin-0.12.1/
bitcoin-0.12.1/bin/
bitcoin-0.12.1/bin/bitcoin-cli
bitcoin-0.12.1/bin/bitcoind
bitcoin-0.12.1/bin/bitcoin-qt
bitcoin-0.12.1/bin/bitcoin-tx
bitcoin-0.12.1/bin/test_bitcoin
bitcoin-0.12.1/include/
bitcoin-0.12.1/include/bitcoinconsensus.h
bitcoin-0.12.1/lib/
bitcoin-0.12.1/lib/libbitcoinconsensus.so
bitcoin-0.12.1/lib/libbitcoinconsensus.so.0
bitcoin-0.12.1/lib/libbitcoinconsensus.so.0.0.0
```

I think we should add a README.md with the release that describes how to get started, and what every file is for.

For windows we have `doc/README_windows.txt` which gets packaged as part of the installer. It is not packaged as part of the windows zips though.

### LINKED ISSUE #8161: Binary release doesn't make it easy to find the source code
This has been suggested before, but I cannot find an issue for it.

I think we should include a link to this repository in various places. At least:
- [x] The "About" dialog in the GUI [done in #8207]
- [ ] A README that comes with release tarballs (see #8160)

This makes it easier to find the source code for potential contributors, as well as avoids awkward questions like #8157.

### PR DESCRIPTION
**Pull request by cisba**
- README.md main file placed inside share/doc/bitcoin lsb directory
    - Updated README.md to include link to repo, files list and description
    - Added example bitcoin.conf, init files for daemon, RPCAuth Tool

    This fixes #8160 (partially), fixes #8161, supersedes #14026, related also to #11971 and #14022

### CODE REVIEW

**Review by randolf — APPROVED** [review]

File: contrib/gitian-descriptors/gitian-linux.yml [hunk 1]
```diff
@@ -181,9 +181,15 @@ script: |
     rm -rf ${DISTNAME}/lib/pkgconfig
     find ${DISTNAME}/bin -type f -executable -exec ../contrib/devtools/split-debug.sh {} {} {}.dbg \;
     find ${DISTNAME}/lib -type f -exec ../contrib/devtools/split-debug.sh {} {} {}.dbg \;
-    cp ../doc/README.md ${DISTNAME}/
     find ${DISTNAME} -not -name "*.dbg" | sort | tar --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ${OUTDIR}/${DISTNAME}-${i}.tar.gz
     find ${DISTNAME} -name "*.dbg" | sort | tar --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ${OUTDIR}/${DISTNAME}-${i}-debug.tar.gz
+    DOCDISTNAME="${DISTNAME}/share/doc/bitcoin"
+    mkdir -p ${DOCDISTNAME}/init
+    cp ../doc/README.md ${DOCDISTNAME}
```
Comment (laanwj) [inline contrib/gitian-descriptors/gitian-linux.yml]: I think the goal of putting the README.md in the root was to make it easy to find for someone extracting it

If you move it to `share/doc/bitcoin`, well, you might as well make it part of the general `make install` logic in the build system.

**Review by DocOBITCOIN — APPROVED** [review]

File: contrib/gitian-descriptors/gitian-linux.yml (hunk 1 above)
Comment (cisba) [reply contrib/gitian-descriptors/gitian-linux.yml]: I included doc/README.md in the `make install` logic

**Comment by fanquake** [comment]
@cisba What's the status of this PR, is it still a work in progress? Can you squash to remove the merge commits.

**Comment by fanquake** [comment]
Closing "Up for grabs". Comment if you'd like this re-opened.

**Comment by cisba** [comment]
> @cisba What's the status of this PR, is it still a work in progress? Can you squash to remove the merge commits.

Hi @fanquake ,
I restarted working to this issue with automake to include the documentation in the "make install" logic

The squash you required is done on my branch

**Comment by cisba** [comment]
> Closing "Up for grabs". Comment if you'd like this re-opened.

Sorry for delay. Family and job came first. :)

**Comment by cisba** [comment]
Please @fanquake re-open it.
Thanks

**Comment by MarcoFalke** [comment]
@cisba just open a fresh pull request. This one cannot be reopened because the branch has been rewritten (force-pushed)

**Comment by cisba** [comment]
> @cisba just open a fresh pull request. This one cannot be reopened because the branch has been rewritten (force-pushed)
Done with PR #15554

**Pull request closed without merging**